tools/ed64romconfig.c

```diff
@@ -51,9 +67,23 @@ int print_usage(const char * prog_name)
 	fprintf(stderr, "See: https://github.com/krikzz/ED64/blob/master/docs/rom_config_database.md#developer-override\n");
 	fprintf(stderr, "\n");
 	fprintf(stderr, "Command-line flags:\n");
-	fprintf(stderr, "\t-w, --savetype <type>  Declare cartridge save type.\n");
-	fprintf(stderr, "\t-c, --rtc              Declare real-time clock support.\n");
-	fprintf(stderr, "\t-r, --regionfree       Declare region-free ROM.\n");
+	fprintf(stderr, "\t-w, --savetype <type>           Declare cartridge save type.\n");
+	fprintf(stderr, "\t-c, --rtc                       Declare real-time clock support.\n");
+	fprintf(stderr, "\t-r, --regionfree                Declare region-free ROM.\n");
+	fprintf(stderr, "\t-C1, --controller1 <type>       Define controller 1 hardware type. <type> should be one of:\n");
+	fprintf(stderr, "\t    default                     Emulators should follow standard config for this port.\n");
+	fprintf(stderr, "\t    n64,pak=rumble              N64 controller with Rumble Pak\n");
+	fprintf(stderr, "\t    n64,pak=controller          N64 controller with Controller Pak\n");
+	fprintf(stderr, "\t    n64,pak=transfer            N64 controller with Transfer Pak\n");
+	fprintf(stderr, "\t    nothing                     Nothing attached to this port\n");
```

Comment:
I can think of a couple other aliases that I would try before "nothing" if I wasn't sure of the syntax:
* none
* empty
* disconnected

My gut feeling based on nothing is that "none" is the right one if we had to pick one (but do we have to pick one?)